wordpress-mobile/WordPressAuthenticator-iOS
# UL&S: Add TextFieldTableViewCell to SiteAddressViewController

Ref. #283 

This PR adds a basic textfield inside a tableview cell. The cell should look like the Zeplin designs. This PR also abstracts out the row presentation.

### To test
1. `bundle exec pod install`
2. Build and run
3. Check out the testing PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/14340

<img width="351" alt="Screen Shot 2020-06-17 at 11 29 01 AM" src="https://user-images.githubusercontent.com/1062444/84924164-c7f7bb00-b08d-11ea-8a16-9d9b7672ce5a.png">

**iPhone 11 Pro screenshots**
<a href="https://user-images.githubusercontent.com/1062444/85311821-9e6ad500-b47b-11ea-9c16-e289ca1d524c.png"><img src="https://user-images.githubusercontent.com/1062444/85311821-9e6ad500-b47b-11ea-9c16-e289ca1d524c.png" width="350" /></a>

<a href="https://user-images.githubusercontent.com/1062444/85311854-aaef2d80-b47b-11ea-832b-1e7770c125f4.png"><img src="https://user-images.githubusercontent.com/1062444/85311854-aaef2d80-b47b-11ea-832b-1e7770c125f4.png" width="350" /></a>
